docs/source/index.rst

```diff
@@ -5,7 +5,7 @@
 
 
 ===========================================
-asyncbleach: The async transformation code.
+unasync: The async transformation code.
 ===========================================
```

Comment:
You need to remove a few `=` here to make sure index.rst is still valid

Reply:
I have updated the index.rst